docs/HowToFAQ/FAQSplitHere.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+---
+
+## Question
+
+How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+
+## Answer
+
+Analysis of a method for the purpose of verification identifies a large number of assertions that must be proved.
+These assertions are grouped into batches.
+
+One default way of operating is to combine all assertions into one batch, make one verification condition out of it,
+and have the prover tackle the whole thing at once. This method allows the prover to use intermediate results to 
+more efficiently prove all of the assertions. However, even when this is more efficient than the combination of proving each
+assertion by itself (with start-up costs and the like for each one), it can also take quite a while to give a final result, possibly even timing-out.
```

Comment:
"it can also take quite a while to give a final result", the situation is worse than just the total solving time becoming bigger right? The solving time of one assertion may also become bigger because of the presence of another one, because there is more "clutter" in the solver's state.

Reply:
Possibly, but it is also definitely the case that two different assertions can benefit from the prover internally establishing (internal) lemmas etc. just once, in a combined VC attempt. At least the original engineering folklore for provers was that it was usually in total more efficient to attempt all assertions at once -- because of similar work needing to be done only once. However, I don't know if that has been measured recently with current designs of proof tools. It would be a nice exercise.